### PR TITLE
Header & Footer changes

### PR DESCRIPTION
**Pull request by aeswibon**
✅ Re-wrote the entire Footer component.
✅ Made some changes in the Header component (added effect after user login).
✅ Added some new routes.
✅ Added Reset and Forgot Password form.

### CODE REVIEW

File: acm_website/src/Components/UI/Footer.js [hunk 1]
```diff
@@ -1,71 +1,68 @@
...
+                <li className="mb-2"><a href="#!" className="text-white">Forum</a></li>
+                <li className="mb-2"><a href="#!" className="text-white">Complier</a> </li>
+              </ul>
+            </div>
+            <div className="col-md-4 col-lg-3 col-xl-3 mx-auto mb-md-0 mb-4">
+              <h6 className="text-uppercase fw-bold">Contact</h6>
+              <hr className="mt-0 d-inline-block mx-auto" style={{ width: "60px", backgroundColor: "#fff", height: "2px" }} />
+              <ul className="list-unstyled">
+                <li><i className="fas fa-home mr-3 mb-3"></i> ACM VNIT</li>
+                <li><i className="fas fa-envelope mr-3 mb-3"></i> proclubvnit @gmail.com</li>
```
Comment (sarthakakre27) [inline acm_website/src/Components/UI/Footer.js]: hyperlink mailto email

File: acm_website/src/Components/UI/Footer.js [hunk 2]
```diff
@@ -1,71 +1,68 @@
...
+                <li><i className="fas fa-phone mr-3 mb-3"></i> +91 8830036413</li>
```
Comment (sarthakakre27) [inline acm_website/src/Components/UI/Footer.js]: remove number

File: acm_website/src/Components/UI/header/header.js [hunk 3]
```diff
@@ -1,111 +1,71 @@
...
+import { useSelector, useDispatch } from "react-redux";
+import { selectUser, logout } from "../../../app/userReducer";
 const Header = () => {
-  // States variable to handle the opening and closing of header in mobile view
-  const [isOpen, setOpen] = useState(false); // toggle the header btn in mobile view
+  const user = useSelector(selectUser);
+  const dispatch = useDispatch();
+  const logOut = () => {
+    dispatch(logout());
+    window.location.href = "/home";
```
Comment (sarthakakre27) [inline acm_website/src/Components/UI/header/header.js]: use react router to navigate to "/"

File: acm_website/src/Components/Login/Reset Password.js [hunk 4]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const Reset = () => {
+  return (
+    <>
+     Hello
```
Comment (sarthakakre27) [inline acm_website/src/Components/Login/Reset Password.js:6]: where is form?

**Comment by aeswibon** [comment]
I have made all the changes and resolved any conflict.

**Pull request merged**